website/docs/d/projects.html.markdown

```diff
@@ -61,6 +61,7 @@ data "mongodbatlas_projects" "test" {
 * `org_id` - The ID of the organization you want to create the project within.
 * `cluster_count` - The number of Atlas clusters deployed in the project.
 * `created` - The ISO-8601-formatted timestamp of when Atlas created the project.
+* `tags` - Map that contains key-value pairs between 1 to 255 characters in length for tagging and categorizing the cluster. To learn more, see [Resource Tags](https://dochub.mongodb.org/core/add-cluster-tag-atlas)
```

Comment:
```suggestion
* `tags` - Map that contains key-value pairs between 1 to 255 characters in length for tagging and categorizing the project. To learn more, see [Resource Tags](https://dochub.mongodb.org/core/add-cluster-tag-atlas)
```

Reply:
link seems wrong - please make sure you check carefully after copy/pasting

Reply:
Thank you. Good catch. 
Fixed in 51a0496a14a971c51daebda9bef68575e288dead
Also, added a step on my personal PR review checklist ✅